docs/reference/design/endpoints.md

```diff
@@ -238,8 +235,21 @@ Having the appropriate `canonical_path` allows you to both generate and parse hr
   * `EndpointDefinition.to_href(<named arguments hash>)` to generate an href for the resource.
   * `EndpointDefinition.parse_href(<href String>)` to get a type-coerced hash of the parameters for the canonical action from the given string.
 
-Given a controller (class or instance), you can use use those helpers by first calling its its `definition` method to retrieve the `EndpointDefinition` it implements, and then using either `to_href` or `parse_href` as described above.
+Given a controller (class or instance), you can use use those helpers by first calling its `definition` method to retrieve the `EndpointDefinition` it implements, and then using either `to_href` or `parse_href` as described above. For example:
 
+```ruby
```

Comment:
I added this example here to clarify how to use them. I am not sure if it's a proper example. But I think we need one

Reply:
👍